src/index.html

```diff
@@ -17,6 +29,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="icone moyo"
+        />
+      </a>
+      <nav>
+        <ul class="menu_box">
+          <li>
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="menu_link is-active"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line before this `li` element.